src/pages/ImportData.js

```diff
@@ -77,40 +77,224 @@ const ResponseFragment = ({ title, snapshotResponses, responseIndex }) => {
   ])
 }
 
-const ImportData = () => {
+const ImportDataOverview = ({ header, snapshots, isDataset, snapshotResponses, url }) => div({ style: styles.card }, [
+  h2({ style: styles.title }, [header]),
+  !_.isEmpty(snapshots) ?
+    div({ style: { marginTop: 20, marginBottom: 60 } }, [
+      'Dataset(s):',
```

Comment:
This code wasn't changed as part of this but personally I don't like UIs that use forms like `(s)`. I think the UI should either count the number and then pluralize appropriately, or always use plural and be wrong the in the case where there's only one entry.

Reply:
I am going to defer this to the future, because I wanted to hold off on making functional changes to the UI in this PR, if that's alright?